<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlatformRename</class>
 <widget class="QDialog" name="PlatformRename">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Name Platform</string>
  </property>
  <widget class="QLineEdit" name="rename_plat_box">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>41</y>
     <width>287</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="rename_plat_lab">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>9</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter a new name for the platform</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rename_plat_ok_but">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rename_plat_ok_but</sender>
   <signal>clicked()</signal>
   <receiver>PlatformRename</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
